zanata-war/src/main/java/org/zanata/action/DashboardAction.java

```diff
@@ -59,7 +60,8 @@
 
 @Name("dashboardAction")
 @Scope(ScopeType.PAGE)
-@Restrict("#{identity.loggedIn}")
+@ZanataSecured
+@CheckLoggedIn
```

Comment:
It will require a different Interceptor or we change ZanataSecurityInterceptor to also accept ```@CheckLoggedIn```. I don't think it will give us much value. Especially it can be put on method and for those cases you still need to remember to put ```@ZanataSecured``` plus changing it once migrated to CDI. I am writing a test to verify the annotation now. 
----- Original Message -----

> From: "Sean Flanigan" <notifications@github.com>
> To: "zanata/zanata-server" <zanata-server@noreply.github.com>
> Cc: "Patrick Huang" <pahuang@redhat.com>
> Sent: Thursday, July 30, 2015 10:48:22 AM
> Subject: Re: [zanata-server] Replace Seam Security (#908)

> In zanata-war/src/main/java/org/zanata/action/DashboardAction.java :
> > @@ -59,7 +60,8 @@
> >
> >  @Name("dashboardAction")
> >  @Scope(ScopeType.PAGE)
> > -@Restrict("#{identity.loggedIn}")
> > +@ZanataSecured
> > +@CheckLoggedIn

> > > * Should we put @Interceptors (ZanataSecurityInterceptor.class) on the
> > > @interface CheckLoggedIn too?
> > 
> 

> > No need to do that.
> 
> But it would guarantee that a class-level @CheckLoggedIn would work, would it
> not? Without having to remember the @ZanataSecured .

> —
> Reply to this email directly or view it on GitHub .